modules/supporting_encapsulation.adoc

```diff
@@ -0,0 +1,93 @@
+  | |-/dev/shm                                                                                                                                             tmpfs                  tmpfs      rw,nosuid,nodev,seclabel
+  | |-/dev/pts                                                                                                                                             devpts                 devpts     rw,nosuid,noexec,relatime,seclabel,gid=5,mode=620,ptmxmode=000
+  | |-/dev/mqueue                                                                                                                                          mqueue                 mqueue     rw,relatime,seclabel
+  | `-/dev/hugepages                                                                                                                                       hugetlbfs              hugetlbfs  rw,relatime,seclabel,pagesize=2M
+  |-/run                                                                                                                                                   tmpfs                  tmpfs      rw,nosuid,nodev,seclabel,mode=755
+  | |-/run/containers/storage/overlay-containers/35f65a7e081f135c3809350240e94cf6113b6e283f34191f8a5976041a52430e/userdata/shm                             shm                    tmpfs      rw,nosuid,nodev,noexec,relatime,context="system_u:object_r:container_file_t:s0:c284,c530",size=65536k
+  | |-/run/utsns/f6f4d4cd-a701-462b-87dd-ec645b49f6b0                                                                                                      nsfs[uts:[4026532370]] nsfs       rw,seclabel
+  | |-/run/ipcns/f6f4d4cd-a701-462b-87dd-ec645b49f6b0                                                                                                      nsfs[ipc:[4026532371]] nsfs       rw,seclabel
+  | |-/run/netns/f6f4d4cd-a701-462b-87dd-ec645b49f6b0                                                                                                      nsfs[net:[4026531992]] nsfs       rw,seclabel
+  | |-/run/containers/storage/overlay-containers/ec566c5b5e84eebe1f2e0f9259ac0096e87f6e49041f3f136379f7efbe4e74f2/userdata/shm                             shm                    tmpfs      rw,nosuid,nodev,noexec,relatime,context="system_u:object_r:container_file_t:s0:c294,c852",size=65536k
+  | |-/run/containers/storage/overlay-containers/ba6dc1dd63e5078fdb46b4a492b2172d63b10d26a4e8fb613ab301658ef3d1b2/userdata/shm                             shm                    tmpfs
+----
+
+To start a new interactive shell inside the Kubernetes namespace, run the `kubensenter` script without any arguments:
+[source,terminal]
+----
+$ sudo kubensenter
+----
++
+.Example output
+[source,terminal]
+----
+kubensenter: Autodetect: kubens.service namespace found at /run/kubens/mnt
+#
```

Comment:
```suggestion
#
----
```
I think this is missing the closing `----`?

Reply:
^^ Tony this is your build failing issue

Reply:
It sure was. Fixed.

Reply:
Fixed.